zstacklib/zstacklib/utils/test_ipv6_utils.py

```
@@ -0,0 +1,116 @@
+import sys
+import unittest
+import socket
+import ipaddress
+from unittest.mock import MagicMock, patch
+
+
+class TestHttpIpv6Bind(unittest.TestCase):
+    """TP-047: CherryPy server.socket_host IPv6 双栈绑定（http.py L308-314）"""
+
+    def test_ipv6_socket_available_binds_colon_colon(self):
+        """TP-047a: 系统支持 IPv6 时，socket_host = '::'"""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)